usaspending_api/spending_explorer/v2/filters/explorer.py

```diff
@@ -81,8 +81,8 @@ def object_class(self):
     def recipient(self):
         # Recipients Queryset
         alt_set = self.alt_set.filter(~Q(transaction_obligated_amount=Decimal('NaN')), award__recipient__isnull=False).\
-            annotate(id=F('award__recipient__legal_entity_id'), type=Value('recipient', output_field=CharField()),
-                     name=F('award__recipient__recipient_name'), code=F('award__recipient__recipient_unique_id')).\
+            annotate(id=F('award__recipient__recipient_name'), type=Value('recipient', output_field=CharField()),
+                     name=F('award__recipient__recipient_name'), code=F('award__recipient__recipient_name')).\
```

Comment:
Why is `award__recipient__recipient_name` used for both `id` and `code`?

Reply:
it's done this way to that the grouping still happens on the recipient name, but also to maintain the response parameters necessary by the frontend. By keeping `id` and `code` as the same column, the grouping isn't affected by varying values.